packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.html

```diff
@@ -1,7 +1,13 @@
 <template>
   <template if:true={shouldDisplay}>
-    <li class={tabClass} title={label} role="presentation">
-      <a class="slds-tabs_default__link" role="tab" onclick={select}>{label}</a>
-    </li>
+    <button
+      class={tabClass}
+      onclick={select}
+      aria-label={arialLabelText}
+      aria-pressed={isActive}
```

Comment:
Make sure to test that when `isActive` is `true`, `aria-pressed` is set to the string `"true"`, and when `isActive` is `false`, `aria-pressed` is set to the string `"false"`. We've had issues in Atomic with Stencil simply adding/removing an empty attribute based on true/false.

Reply:
variable ariaPressed added to ensure attribute aria-pressed receives a string.